src/cascadia/TerminalApp/App.cpp

```diff
@@ -362,6 +406,18 @@ namespace winrt::TerminalApp::implementation
 
             feedbackFlyout.Click({ this, &App::_FeedbackButtonOnClick });
             newTabFlyout.Items().Append(feedbackFlyout);
+
+            // Create the about button.
+            auto aboutFlyout = Controls::MenuFlyoutItem{};
+            aboutFlyout.Text(L"About");
+
+            Controls::FontIcon aboutIco{};
+            aboutIco.Glyph(L"\xF142");

```

Comment:
can this have a name? perhaps in unicode.hpp?

Reply:
This will be helped by using `Symbol::Help()`